src/ripple/rpc/handlers/AccountTx.cpp

```diff
@@ -240,6 +240,31 @@ doAccountTxHelp(RPC::Context& context, AccountTxArgs const& args)
 
     result.ledgerRange = std::get<LedgerRange>(lgrRange);
 
+    // if the account does not exist in any of the ledger range,
+    // return an error
+    if (context.apiVersion > 1u)
+    {
+        bool accountFound = false;
+        for (unsigned int idx = result.ledgerRange.min;
+             idx <= result.ledgerRange.max;
+             ++idx)
+        {
+            std::shared_ptr<ReadView const> ledger;
+            auto const status = getLedger(ledger, idx, context);
```

Comment:
You're not checking `status` or `ledger`. That's probably not a good idea. What happens if the ledger is not found? I'm not super familiar with the surrounding code, so I don't know if ledger gaps are possible by the time you get here, but if they are, you need to check.